frontend/amundsen_application/static/js/components/Table/index.tsx

```diff
@@ -4,14 +4,27 @@
 import * as React from 'react';
 
 import { FormattedDataType } from 'interfaces/ColumnList';
-import {
-  COLUMN_NAME_REGEX,
-  TYPE_METADATA_REGEX,
-} from 'components/Table/constants';
+import { getMaxNestedColumns } from 'config/config-utils';
```

Comment:
Sorry, I need to insist about having this passed as an option instead of getting it from configuration.